ironfish/src/memPool/memPool.test.ts

```diff
@@ -99,26 +99,38 @@ describe('MemPool', () => {
   describe('orderedTransactions', () => {
     const nodeTest = createNodeTest()
 
-    it('returns transactions from the node mempool sorted by fees', async () => {
+    it('returns transactions from the node mempool sorted by fee rate', async () => {
       const { node } = nodeTest
       const { wallet, memPool } = node
       const accountA = await useAccountFixture(wallet, 'accountA')
       const accountB = await useAccountFixture(wallet, 'accountB')
       const accountC = await useAccountFixture(wallet, 'accountC')
-      const { transaction: transactionA } = await useBlockWithTx(node, accountA, accountB)
-      const { transaction: transactionB } = await useBlockWithTx(node, accountB, accountC)
+      const { block, transaction: transactionA } = await useBlockWithTx(
+        node,
+        accountA,
+        accountB,
+      )
+      const transactionB = block.transactions[0]
```

Comment:
I think it'd be worth using the `block.minersFee` helper here -- not a huge deal, but makes it a bit clearer.